compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -455,12 +455,12 @@ public void updateXMLSubtypeOwners() {
     }
 
     public void defineOperators() {
-        // Binary arithmetic operators
-        defineIntegerArithmeticOperations();
-
         // Binary arithmetic operators for nullable integer types
         defineNilableIntegerArithmeticOperations();
 
+        // Binary arithmetic operators
```

Comment:
Why does the order have to change here?

Reply:
Here Instead of `function(int, int?) return int?` operator, we can use `function(int?, int?) return int?` operator
With that, we can remove operators define from L713 to L726

Reply:
Why does the order matter?

Reply:
Fixed